src/site/content/en/blog/responsiveness/index.md

```diff
@@ -14,9 +14,7 @@ tags:
 ---
 
 {% Aside 'important' %}
-A candidate for a new responsiveness metric has been chosen, so feedback is no
-longer necessary at this time. When finalized, this metric will be documented
-with its own landing page. This space will be updated when that happens.
+We have chosen option 2 of [high quantile approximation](/responsiveness/#high-quantile-approximation) as a candidate for the new responsiveness metric, with [maximum event duration](/responsiveness/#measure-interaction-latency) being the definition for event interaction. The `experimental.responsiveness` table in the February 2022 CrUX dataset will include this new metric when it releases on 8 March, 2022.
```

Comment:
A few comments on this line:
- s/a candidate/the candidate/
- s/being the definition for event interaction/as the definition for interaction latency/

Also, nit: I don't think `experimental.responsiveness` is a table, so you might want to rephrase that as something like: "This metric will be available as `experimental.responsiveness` starting  in the February 2022 [CrUX BigQuery](https://developers.google.com/web/tools/chrome-user-experience-report/bigquery/getting-started) dataset, released on March 8, 2022.